src/models/gridOption.interface.ts

```diff
@@ -108,6 +108,9 @@ export interface GridOption<C extends BaseColumn = BaseColumn> {
   /** Default cell Formatter that will be used by the grid */
   defaultFormatter?: Formatter;
 
+  /** Escape hatch geared towards testing Slickgrid in jsdom based environments to circumvent the lack of stylesheet.ownerNode and clientWidth calculations */
+  devMode?: false & { ownerNodeIndex?: number; containerClientWidth?: number; };
```

Comment:
isn't that supposed to be `|` instead of `&`, because it can never be both at the same time

Reply:
its an intersection, so a type being either false or object. with an union you'd get a compile error due to false and obj not sharing enough (false does not exist on tyoe ...)

Reply:
I also tried it on my side and that worked with `|` but ok if you're sure then let's merge. Thanks for confirming. I'll copy that code to next slickgrid-universal too since I'll soon have my own slickgrid/dataview in my repo